x-pack/packages/kbn-elastic-assistant/impl/assistant/settings/translations.ts

```diff
@@ -21,6 +21,20 @@ export const SETTINGS_TOOLTIP = i18n.translate(
   }
 );
 
+export const SETTINGS_UPDATED_TOAST_TITLE = i18n.translate(
+  'xpack.elasticAssistant.assistant.settings.settingsUpdatedToastTitle',
+  {
+    defaultMessage: 'Settings updated',
+  }
+);
+
+export const SETTINGS_UPDATED_TOAST_TEXT = i18n.translate(
+  'xpack.elasticAssistant.assistant.settings.settingsUpdatedToastText',
+  {
+    defaultMessage: 'Reload the page to apply the changes',
```

Comment:
But I don't want to reload the page! 😅 Will see if I can see a way around w/ the settings updater... 🙂

Reply:
thanks for your help to remove this requirement! 🙏